src/js/components/FormField/FormField.js

```diff
@@ -86,6 +87,7 @@ class FormFieldContent extends Component {
         }}
         plain
         focusIndicator={false}
+        aria-invalid={invalid || undefined}
         {...rest}
```

Comment:
If the user can leverage the `...rest` to set `aria-invalid`, why do grommet needs to add it manually?

Reply:
Because grommet is managing the validation. We will set `aria-invalid` based on our evaluation of the caller supplied regex or validation function result.